lib/files.js

```diff
@@ -305,9 +334,11 @@ exports.getComponentPackage = control.memoize(getComponentPackage);
 exports.isDirectory = control.memoize(isDirectory);
 exports.tryRequire = control.memoize(tryRequire);
 exports.readFilePromise = control.memoize(readFilePromise);
+exports.getFilesPromise = control.memoize(getFilesPromise);
```

Comment:
`readFilePromise` returns the contents of a file, `getFilesPromise` is like `getFiles` in that it returns all the filenames within a directory. I just needed to have a version of `getFiles` which returned a Promise.